src/esp/assets/Asset.cpp

```diff
@@ -14,12 +14,17 @@ using namespace Corrade::Utility::String;
 AssetInfo AssetInfo::fromPath(const std::string& path) {
   AssetInfo info{AssetType::UNKNOWN, path};
 
-  if (endsWith(path, "semantic_quad_mesh.ply")) {
+  if (endsWith(path, "mesh_semantic.ply")) {
```

Comment:
I just restored the FRL_INSTANCE_MESH.
WARNING: It will segfault now.
The old code somehow cannot handle the new replica instance model.
This put me in a dilemma: 
a) fix the segfault;
b) retire the FRL_INSTANCE_MESH directly in the future;

Reply:
No no no.  `semantic_quad_mesh.ply` was the correct ends with.  That mesh format is not for the replica meshes. See my other comment.

Reply:
@erikwijmans : Then this is the old format with the SurrealApt, and I do not think we need to support it any more.
Shall we retire this type?

Reply:
Yes